package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "web3-react-native-dapp-wagmi",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "@react-native-async-storage/async-storage": "1.18.2",
+    "@react-native-community/netinfo": "9.3.10",
+    "@web3modal/core-react-native": "^1.0.0-alpha.12",
+    "@web3modal/wagmi-react-native": "^1.0.0-alpha.11",
+    "expo": "~49.0.15",
+    "expo-status-bar": "~1.6.0",
+    "fastestsmallesttextencoderdecoder": "^1.0.22",
+    "react": "18.2.0",
+    "react-native": "0.72.6",
+    "react-native-get-random-values": "~1.9.0",
+    "react-native-modal": "^13.0.1",
+    "react-native-svg": "13.9.0",
+    "viem": "^1.16.6",
+    "wagmi": "^1.4.5"
+  },
```

Comment:
Q: do we need any crypto package or is that imported indirectly by Viem/Wagmi?

Reply:
We use `react-native-get-random-values`, per the WalletConnect docs